regolith/builders/grantreportbuilder.py

```diff
@@ -90,13 +91,32 @@ def latex(self):
                                      types=["all"], since=rp_start_date, before=rp_end_date, statuses=["accepted"]))
         # thesis defendings
         # how do i access people.yml in rg-db-public vs the people.yml file in rg-db-group?
+        defended_theses = []
         for id in grant_people:
             person = self.gtx['people'][id]
-            if person['']
-        # How have results been disseminated
+            for education in person['education']:
+                if 'phd' in education['degree'].lower() and 'columbia' in education['institution'].lower() and \
+                        rp_start_date.year <= education['end_year'] <= rp_end_date.year:
+                    defended_theses.append(id)
 
-        # Plans for Next Reporting Period to Accomplish Goals
+        # Products
+        # need rg-db-public's citation.yml... how to access that instead of the rg-db-group's citation.yml file
+        publications = filter_publications(self.gtx["citations"], grant_people, since=rp_start_date, before=rp_end_date)
 
+        # Participants/Organizations
+        participants = {}
+        for person in grant_people:
+            p = self.gtx["people"].get(person)
```

Comment:
please don't be lazy with variable names, it makes the code hard to read.  What is p?

Here I think the logic is that p contains the person document for the person of name person, so I would pick variable names like this:
```
for name in grant_people:
    person = self.gtx["people"].get(person)
```
I think this logic assumes that the names in grant_people are valid id's is that right?  This might be a bit brittle.